sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -95,10 +95,13 @@ class DataSourceV2Strategy(session: SparkSession) extends Strategy with Predicat
   }
 
   private def makeQualifiedNamespacePath(location: String): String = {
-    val warehousePath = session.sharedState.conf.get(WAREHOUSE_PATH)
-    val nsPath = CatalogUtils.makeQualifiedNamespacePath(
-      CatalogUtils.stringToURI(location), warehousePath, session.sharedState.hadoopConf)
-    CatalogUtils.URIToString(nsPath)
+    CatalogUtils.makeQualifiedLocationPath(session.sharedState.conf.get(WAREHOUSE_PATH),
```

Comment:
should we use a more generic method name and merge `makeQualifiedNamespacePath` and `makeQualifiedTablePath`

Reply:
Thank you for your advice. It would be better if merge them to one. How about `makeQualifiedDBObjectPath`?

Reply:
I think `makeQualifiedDBObjectPath ` is OK